src/webviews/webviewBase.ts

```diff
@@ -50,7 +51,10 @@ export abstract class WebviewBase<State> implements Disposable {
 		showCommand: Commands,
 	) {
 		this._originalTitle = this._title = title;
-		this.disposables.push(registerCommand(showCommand, this.onShowCommand, this));
+		this.disposables.push(
+			registerCommand(showCommand, this.onShowCommand, this),
+			window.onDidChangeWindowState(this.onWindowStateChanged, this),
```

Comment:
We should move this down into the `show` method when we are setting up event handlers for the webview only when its visible

Reply:
Done in commit: `Moved "onDidChangeWindowState" code down into the show method `